GitForWindowsHelper/index.js

```diff
@@ -72,6 +72,17 @@ module.exports = async function (context, req) {
         return withStatus(500, undefined, e.message || JSON.stringify(e, null, 2))
     }
 
+    try {
+        const {renameCurrentAndCreateNextMilestone} = require('./update-milestones')
+        if (req.headers['x-github-event'] === 'pull_request'
+            && req.body.repository.full_name === 'git-for-windows/git'
+            && req.body.action === 'opened'
+            && req.body.pull_request.merged === 'true') return ok(await renameCurrentAndCreateNextMilestone(context, req))
+    } catch (e) {
+        context.log(e)
+        return withStatus(500, undefined, e.message || JSON.stringify(e, null, 2))
+    }
+
```

Comment:
I'd rather do that when publishing the final release, i.e. at the same time as the current milestone is closed.

Reply:
That would prevent issues with late component updates.

Reply:
Isn't it the other way round? When a final rebase PR is opened, and `/git-artifacts` produces an installer that has issues that require a `component-update` ticket to be addressed, we'd still want that in the same milestone, right?